src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -77,10 +77,10 @@ const NotebookPreviewFrame = ({ notebookName, workspace: { workspace: { namespac
   return h(Fragment, [
     preview && h(Fragment, [
       div({ style: { position: 'relative' } }, [
-        linkButton({
+        buttonPrimary({
           style: { position: 'absolute', top: 20, left: 'calc(50% + 580px)' },
-          href: Nav.getLink('workspace-notebooks', { namespace, name })
-        }, [icon('times-circle', { size: 30 })])
+          onClick: () => Nav.goToPath('workspace-notebooks', { namespace, name })
+        }, ['Close', icon('times-circle', { size: 30, style: { marginLeft: '0.5rem' } })])
```

Comment:
I think the times-circle looks really big/heavy right now. It might be a good idea to just remove it entirely since it's redundant with the "Close" text.

Reply:
Sounds good! Removed the `x`